src/filler.ts

```diff
@@ -40,7 +41,7 @@ export default <N extends Node<Raw>>(rawNode: N) => FilledBehavior<NodeOfKind<Ki
       ...transformed,
       superclass: transformed.name === 'Object' ? null : transformed.superclass ?? OBJECT_CLASS,
       members: [
-        ...transformed.members.some(member => member.kind === 'Constructor') ? [] : [DEFAULT_CONSTRUCTOR],
+        ...transformed.name === 'Object' ? [DEFAULT_CONSTRUCTOR] : [],
```

Comment:
I still don't have a feeling of wollok-ts implementation, I only can advise having `'Object'` as a constant (and I don't know the difference with `OBJECT_CLASS` constant)

Reply:
We might make a cross-refactor to extract these into a constants file in the future, but most of those cases (including this line) will be deleted once we drop constructors.